### ISSUE TITLE
Add letter_opener in dev

### ISSUE THREAD

**Issue by patbenatar** [philosophie/suspenders#47]

**Comment by patbenatar** [comment]
@jdwolk you mentioned a preference for `mailcatcher` in the feature analysis spreadsheet.. can you elaborate? I've had more problems than not recently with `mailcatcher` and found `letter_opener` to be an adequate replacement that "just works"

**Comment by taboularasa** [comment]
I agree with nick, mail catcher was broken last I checked, problem with conflicting dependencies or something

**Comment by jdwolk** [comment]
@taboularasa @patbenatar I just hooked up recently on a project and it wasn't broken w/ any of the dependencies there.

I kind of like mailcatcher because it persists mails after they're sent + it's not invasive (i.e. popping up browser tabs and stealing focus).

Granted it's a few lines more config + you have to run a separate process, but that can be daemonized / tmuxified etc

That said I've got no particular preference for one or the other because seems like benefits equally outweigh drawbacks, so I'm cool w/ whatever everyone wants to go with

**Comment by taboularasa** [comment]
@jdwolk one thing I like more with `letter_opener` is it works without background queue running which is a nice to have for designers

**Comment by patbenatar** [comment]
@taboularasa @jdwolk thanks for chiming in! My vote remains for `letter_opener`...
